ravindu14/SliitSpmProject
# [1158] Email sending interface is done

How to Test
  1.Switch to branch feature/task-1158-validate-the-input-field-and-the-email-button
  2. Go to tab Email Form1 in the sidebar
  3. Then the interface Email Form1 is displayed
